sw.js

```diff
@@ -0,0 +1,85 @@
+// Subhankar Pal | @subho57
```

Comment:
It will be better to not have service worker at this point. 

Though Service Workers come with great features, it gets difficult to test sites on localhost and we will have to bump the cache version on every commit for the site to work properly in the production which is difficult when the project is open-source since a lot of times I can just merge PR and expect the changes in production but with this I will have to set up the PRs locally, bump the version, then push to the PR and then merge them which can be tedious.

Also, on this site, there are more new visitors than returning visitors so for the large part of the audience, it will be unnecessary to load the complete site in their device.

Reply:
yeah cool, that makes sense to me. I'll do the changes.